src/modules/mc_hover_thrust_estimator/MulticopterHoverThrustEstimator.cpp

```diff
@@ -105,76 +113,104 @@ void MulticopterHoverThrustEstimator::Run()
 
 	perf_begin(_cycle_perf);
 
-	vehicle_land_detected_s vehicle_land_detected;
-	vehicle_status_s vehicle_status;
-	vehicle_local_position_s local_pos;
+	if (_vehicle_land_detected_sub.updated()) {
```

Comment:
Is this version faster?

Reply:
In this case I was simply limiting the scope of the possibly uninitialized `vehicle_land_detected`.